docker-compose.yml

```diff
@@ -18,13 +18,13 @@ services:
     healthcheck:
       test: [
         "CMD-SHELL",
-        "bh=$(curl -sf http://localhost:26657/status | jq -r '.result.sync_info.latest_block_height'); if [ \"$$bh\" -ge 1 ]; then exit 0; else exit 1; fi"
+        "bh=$(curl -sf http://localhost:26657/status | jq -r '.result.sync_info.latest_block_height'); if [ \"$$bh\" -gt 1 ]; then exit 0; else exit 1; fi"
```

Comment:
Quite a subtle one here, but we must use `-gt` rather than `-ge`. 
Using greater than, ensure that we have a committed and produced the first block. This can then report the service as healthy and allow celestia-node to start up successfully. Otherwise, I was running into a `nil commit for block hash`, as the bridge node started prematurely.